charts/argo-rollouts/values.yaml

```diff
@@ -43,6 +43,8 @@ global:
   deploymentAnnotations: {}
   # -- Labels for all deployed Deployments
   deploymentLabels: {}
+  # -- Number of old deployment ReplicaSets to retain. The rest will be garbage collected.
+  revisionHistoryLimit: 10
```

Comment:
Now default is `10` 📝 
Ref: https://kubernetes.io/docs/reference/using-api/deprecation-guide/

<img width="597" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/5cfeb3d4-c997-4fc7-bf3e-040507f0d695">